### PR TITLE
doc: add workaround for using unity ui on background

### PR DESCRIPTION
**Pull request by kannan-xiao4**
related Issue https://github.com/Unity-Technologies/UnityRenderStreaming/issues/201

### CODE REVIEW

File: com.unity.renderstreaming/Documentation~/input.md [hunk 1]
```diff
@@ -106,3 +106,51 @@ You can control the UnityUI placed on the game scene from browser.
...
+
+### Workaround for using UnityUI on RunInBackground
+
+If use UnityUI in the background, follow the steps below.
+
+1. Update `com.unity.inputsystem` to `1.1.0-preview2` on PackageManager.
+
+2. In ProjectSetting->Player->OtherSetting, check `Allow unsafe code`.
+
+3. Replace the floowing Component with `EventSystem` on the Scene.
```
Comment (sindharta-tanuwijaya) [inline com.unity.renderstreaming/Documentation~/input.md:119]: ~Replace the floowing Component with `EventSystem` on the Scene.~
Replace `EventSystem` in the Scene with the following component.

**Review by sindharta-tanuwijaya — COMMENTED** [review]
Added comment

File: com.unity.renderstreaming/Documentation~/input.md [hunk 2]
```diff
@@ -106,3 +106,51 @@ You can control the UnityUI placed on the game scene from browser.
 > [!NOTE]
 > If the Unity application is in Background, even if the `RunInBackground` checkbox is On, you can not control the UnityUI from browser. This issue will be fixed in `com.unity.inputsystem`.
 
+
+
+### Workaround for using UnityUI on RunInBackground
+
+If use UnityUI in the background, follow the steps below.
+
+1. Update `com.unity.inputsystem` to `1.1.0-preview2` on PackageManager.
```
Comment (karasusan) [inline com.unity.renderstreaming/Documentation~/input.md:115]: `1.1.0-preview2` -> `1.1.0-preview.2`

File: com.unity.renderstreaming/Documentation~/input.md [hunk 3]
```diff
@@ -106,3 +106,51 @@ You can control the UnityUI placed on the game scene from browser.
...
+
+2. In ProjectSetting->Player->OtherSetting, check `Allow unsafe code`.
```
Comment (karasusan) [inline com.unity.renderstreaming/Documentation~/input.md:117]: ProjectSetting -> `Project Settings` 
OtherSetting -> `Other Settings`
Allow unsafe code -> `Allow 'unsage' Code`

File: com.unity.renderstreaming/Documentation~/input.md [hunk 4]
```diff
@@ -106,3 +106,51 @@ You can control the UnityUI placed on the game scene from browser.
...
+    }
+
+    protected override void OnApplicationFocus(bool hasFocus)
+    {
+        //Do not change focus flag on eventsystem
+    }
+}
+```
+
+4. (Optional) For UnityEditor, Open Window->Analysis->InputDebugger and turn on `Lock Input to Game View` in Options.
```
Comment (karasusan) [inline com.unity.renderstreaming/Documentation~/input.md:156]: InputDebugger -> `Input Debugger`

**Review by karasusan — APPROVED** [review]

**Pull request merged**